topostats/io.py

```diff
@@ -92,7 +93,7 @@ def __init__(
         self.image = None
         self.pixel_to_nm_scaling = None
 
-    def load_spm(self) -> tuple:
+    def _load_spm(self) -> tuple:
```

Comment:
I'd purposefully not made this an internal/private method (which is denoted by starting function/method names with `_`) because I figured that people using Notebooks may want to `LoadScan().load_spm()` directly.

Reply:
Ah fair enough, my bad. I'll adjust that now for this and ibw.

I was trying to make it follow the same formatting as you had for the JPK ones.